pubspec.yaml

```diff
@@ -21,9 +21,13 @@ dependencies:
   simple_html_css: ^5.0.0
   webview_flutter: ^4.8.0
   package_info_plus: ^8.1.0
+  build: ^2.4.1
```

Comment:
this can be 2.4.12

Reply:
I only see 2.4.1 as the latest: https://pub.dev/packages/build